hermetic_build/library_generation/cli/entry_point.py

```diff
@@ -62,6 +63,7 @@ def main(ctx):
 )
 @click.option(
     "--api-definitions-path",
+    "--api-root",
```

Comment:
I think `api-root` is the same as `api-definitions-path` since it is the location of googleapis repo.